ytfzf

```diff
@@ -756,24 +743,24 @@ parse_opt () {
 		L) info_to_print="$info_to_print,L" ;;
 		c) scrape=$optarg ;;
 		I) info_to_print=$optarg ;;
-		v) # set variable
+		"$long_opt_char") # set variable
```

Comment:
i don't think this would work, the long_opt_char would be filtered out before parse_opt

Reply:
there is a check to make shre long_opt_char is not alphanumeric or v, v will be reserved as the only alphanumeric char for a long opt